content/guidelines/reviewing.en.md

```diff
@@ -0,0 +1,109 @@
+This is where volunteer reviewers come in to help. Reviewers are people who have volunteered to review contributions made to the project. They are not paid for their work, but they are given credit for their work.
+
+## Who can do it?
+
+Anyone can review contributions. As hinted before, most of the contributions made to Open Terms Archive are contractual documents that are to be tracked. Therefore, you don't need to be a programmer or have any technical knowledge. You just need to be able to read and understand the contribution and provide feedback where necessary.
+
+## How long and complex will it be?
+
+It depends on the contribution. Some contributions may be spot on and can be reviewed in a few minutes. Other contributions may require a more detailed review and changes to be made and can take longer.
+However we approximate it to take about 3-15 mins for one to review a document, and getting better with time.
+
+## Where to start
+
+To get started, we will need to understand a number things. The nature of the contributions you will be reviewing, where to get the contributions to review and the tools that will help you in reviewing the contributions.
+
+### The nature of the contributions
+
+The contributions you will be reviewing are contractual documents of digital services. These are documents that govern the relationship between two or more parties.
+They are not the original documents, but rather accurate tracking and data about these documents. If the documents are represented accurately, it will be easier to follow on with any subsequent changes in the document. Contributors track these documents (sometimes, anonymously) by submitting a pull request either using the **GUI contributing tool** (A tool that helps contributors add documents to the project), or creating a JSON file and adding it via a pull request. You can find more information about pull requests [here](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+
+There are three types of contributions you'll come across:
+
+- Adding contributions to declarations
+- Updating contributions to declarations
+- Removal contributions to declarations
+
+The method used to review each of these types of contribution varies, and you'll find a detailed explanation of this as you proceed with this document.
+
+### Where to find the contributions
+
+These contributions are housed in the [`contrib-declarations`](https://github.com/OpenTermsArchive/contrib-declarations) repository under the pull requests tab. You can find the pull requests [here](https://github.com/OpenTermsArchive/contrib-declarations/pulls).
+
+## How To Review Pull Requests that Add Declarations
+
+When a contribution is made using the contribution tool, the OTA-bot will create a pull request. This pull request typically consists of several checks, which guides the reviewer on the key things to look out for during the review process, and a link to inspect the declaration.
+
```

Comment:
Yeah
When a document is submitted, it's important that the person submitting it knows what to expect. That is why I added this. This is especially important for newbies or those who are unfamiliar with code. Seeing their declaration reflected in a pull request opened by a bot can assure them that they did the right thing and everything is in order. However, my previous description may have been too detailed. Perhaps we can come up with a better way to explain the key concepts and not the current circumstance. 

But thinking this through, this will be more important to contributors than reviewers. It can be mentioned in an onboarding maybe. Because we assume reviewers will already be conversant with contributing